src/assets/custom/css/custom-softmod-feature.scss

```diff
@@ -0,0 +1,751 @@
+    }
+
+    .bg-business-alignment {
+        background-image: url('{{site.baseurl}}/assets/custom/img/softmod/bg-business-alignment.svg');
+        background-position: center bottom 40px;
+        background-repeat: no-repeat;
+        background-size: 50% auto;
+    }
+
+    .bg-people-culture {
+        background-image: url('{{site.baseurl}}/assets/custom/img/softmod/bg-people-culture.svg');
+        background-position: center bottom 50px;
+        background-repeat: no-repeat;
+        background-size: 60% auto;
+    }
+
+    .bg-risk-management {
+        background-image: url('{{site.baseurl}}/assets/custom/img/softmod/bg-risk-management.svg');
+        background-position: center bottom 50px;
+        background-repeat: no-repeat;
+        background-size: 50% auto;
+    }
+
+    .bg-testimonials {
+        background:url('{{site.baseurl}}/assets/custom/img/softmod/bg-testimonials.jpg');
+        background-size: 110% auto;
+        background-position-y: 360px;
+        background-repeat: no-repeat;
+    }
+}
+
+.bg-sheen {
+    position:relative;
+}
+
+.bg-sheen:before {
+    content:'';
+    background: rgb(252,249,241);
+    background: linear-gradient(34deg, rgba(252,249,241,1) 52%, rgba(252,244,244,1) 100%);
+    position: absolute;
+    width:100%;
+    height: 100%;
+    top: 0;
+    z-index: 0;
+    right: 0;
+}
+
+@media (min-width: 1200px) {
+
+    .bg-sheen:before {
+        right: 100px;
+    };
+
+    .bg-sheen:after {
+        content: '';
+        right: 100px;
+        background-color:#293341;
+        position: absolute;
+        width:100%;
+        height: 450px;
+        bottom: 0;
+        z-index: 0;
+        right: 0;
+        background-image: url('{{site.baseurl}}/assets/custom/img/softmod/bg-slate-pattern-2.jpg');
+        background-size: 100% auto;
+        background-repeat: no-repeat;
+    };
+}
+
+/********************************
+    Helpers - Positioning
+********************************/
+
+.z-index-1 {
+    z-index: 1;
+    position: relative;
+}
+
+.p-relative {
+    position: relative;
+}
+
+/********************************
+    Helpers - Containers
+********************************/
+
+.container-slim {
+    padding-left: 15px;
+    padding-right: 15px;
+    margin-left: auto;
+    margin-right: auto;
+}
+
+@media (min-width: 768px) {
+
+    .container-slim {
+        max-width: 720px;
+    }
+}
+
+@media (min-width: 992px) {
+
+    .container-slim {
+        max-width: 800px;
+    }
+}
+
+@media (min-width: 1200px) {
+
+    .container-slim {
+        max-width: 800px;
+    }
+}
+
+/********************************
+    Component - Buttons
+********************************/
+
+.btn--coral {
+    background-color:rgba(251, 114, 104, 1)!important;
+    transition: all 0.3s ease;
+
+    &:hover {
+        background-color:rgba(251, 114, 104, 0.8)!important;
+        transition: all 0.3s ease;
+    }
+}
+
+.btn--blue {
+    background-color:rgba(15, 64, 113, 1)!important;
```

Comment:
Really feels like we should go to the source of the problem rather than just adding more and more overrides to button styles.